aws/README.md

```diff
@@ -31,13 +31,13 @@ Instance type used for all EC2 instances
 Docker version to install on nodes
 
 ###### `rke_kubernetes_version`
-- Default: **`"v1.18.6-rancher1-1"`**
+- Default: **`"v1.18.8-rancher1-1"`**
 Kubernetes version to use for Rancher server RKE cluster
 
 See `rancher-common` module variable `rke_kubernetes_version` for more details.
 
 ###### `workload_kubernetes_version`
-- Default: **`"v1.17.9-rancher1-1"`**
+- Default: **`"v1.17.11-rancher1-1"`**
```

Comment:
Is the available version(s) of k8s for the workload cluster impacted by updating the RKE terraform provider? Or is this PR also adding in updates for the newest release of `rancher/rke`, reflected in the Rancher server's dynamic RKE version list?

Reply:
It's not directly impacted. But because of the dynamic rke version list in Rancher `v1.17.11-rancher1-1` is the current 1.17 default in Rancher 2.4.8.
If you want, I can also split this up in to separate PRs.

<img width="1576" alt="Bildschirmfoto 2020-09-22 um 10 19 01" src="https://user-images.githubusercontent.com/243056/93858973-47b0a980-fcbd-11ea-9d1c-5103b30d7b47.png">

Reply:
Just making sure I know the full scope is all - thanks for the clarification!